languages/en.php

```diff
@@ -2,404 +2,411 @@
 
 return [
 
-	'country:AD' => 'Andorra',
```

Comment:
Your indentation settings are not in line with Elgg coding standards. We use 1 tab